api/swagger.yaml

```
@@ -3357,6 +3357,23 @@ definitions:
             format: "dateTime"
           Message:
             type: "string"
+      ServiceStatus:
+        description: |
+          The status of the service's tasks. Provided only when requested as
+          part of a ServiceList operation.
+        type: "object"
+        properties:
+          RunningTasks:
+            description: "The number of tasks for the service currently in the Running state"
+            type: "number"
+          DesiredTasks:
+            description: |
+              The number of tasks for the service desired to be running.
+              For replicated services, this is the replica count from the
+              service spec. For global services, this is computed by taking
+              count of all tasks for the service with a Desired State other
+              than Shutdown.
+            type: "number"
```

Can you add an example value here, and set the correct format/type?

```suggestion
            type: "integer"
            format: "uint64"
            example: 10
```

---

Can you also add a mention in the API history doc; https://github.com/moby/moby/blob/master/docs/api/version-history.md